build.wake

```diff
@@ -13,100 +13,103 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 package build_wake
 from wake import _
 
 # Useful build variants (arguments to all)
 def toVariant = match _
     "default" = Pass (Pair "native-cpp14-release" "native-c11-release")
-    "static"  = Pass (Pair "native-cpp14-static"  "native-c11-static")
-    "debug"   = Pass (Pair "native-cpp14-debug"   "native-c11-debug")
-    "wasm"    = Pass (Pair "wasm-cpp14-release"   "wasm-c11-release")
+    "static" = Pass (Pair "native-cpp14-static" "native-c11-static")
+    "debug" = Pass (Pair "native-cpp14-debug" "native-c11-debug")
+    "wasm" = Pass (Pair "wasm-cpp14-release" "wasm-c11-release")
     s = Fail "Unknown build target ({s})".makeError
 
+
 export def build: List String => Result String Error = match _
     "tarball", Nil = tarball Unit | rmap (\_ "TARBALL")
     kind, Nil =
         require Pass variant = toVariant kind
         all variant | rmap (\_ "BUILD")
     _ = Fail "no target specified (try: build default/debug/tarball)".makeError
 
+
 export def install: List String => Result String Error = match _
     dest, kind, Nil = doInstall (in cwd dest) kind | rmap (\_ "INSTALL")
     dest, Nil = doInstall (in cwd dest) "default" | rmap (\_ "INSTALL")
     _ = Fail "no directory specified (try: install /opt/local)".makeError
 
+
 # Build all wake targets
 def targets =
-    buildWake, buildWakeBox, buildFuseDaemon, buildShim, buildBSP, buildLSP, buildJobCache, buildWakeFormat, Nil
+    buildWake,
+    buildWakeBox,
+    buildFuseDaemon,
+    buildShim,
+    buildBSP,
+    buildLSP,
+    buildJobCache,
+    buildWakeFormat,
+    Nil
+
 
 def all variant =
-    require Pass x =
-        map (_ variant) targets
-        | findFail
+    require Pass x = map (_ variant) targets | findFail
```

Comment:
I will admittedly sometimes put a pipeline on a single line, but for the most part, I don't think I mind a universal rule of being split over multiple lines. If a pipeline can fit on a single line, it can often just be a function application (though not always). However, there are cases where that doesn't work as well, such as the small unit testing DSL I was using: `triple 3 | should equalInteger 3`. But it wouldn't be the worst thing if that was on multiple lines.

It's also possible that pipelines should not be adjusted, or they follow what we've discussed for other situations, such as for `if then else`.